style.css

```diff
@@ -682,3 +685,84 @@ small {
   font-size: 13px;
   font-family: 'Inter', Arial, Helvetica, sans-serif;
 }
+
+.popTitle {
+  font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  font-size: 20px;
+  margin-left: 0;
+  margin-top: 0;
+}
+
+#project-live-source {
+  margin-left: -40px;
+  justify-content: space-between;
+}
+
+#project-live-source li button a {
+  height: 35px;
+  width: 110px;
+  background: #ff6b00;
+  color: aliceblue;
+  border: rgb(255, 255, 255);
+  font-weight: 700;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+}
+
+#popup {
+  max-width: 45vw;
+}
+
+.popup-close {
+  position: absolute;
+  right: 30px;
+  top: 35px;
+  border: none;
+  background: #fff;
+}
+
+#technologies {
+  margin-left: -40px;
+  column-gap: 10px;
+}
+
+#technologies li button {
+  padding: 5px;
+  border: 1px solid rgb(222, 220, 220);
+  background: white;
+}
+
+.popup-image {
+  width: 100%;
+  height: 220px;
+}
+
+.popup-description {
+  width: 100%;
+  text-align: justify;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 13px;
+  line-height: 19px;
+}
+
+.pop-icon {
+  margin-left: 3px;
+}
+
+.popup {
+  position: fixed;
+  top: 50%;
+  left: 27%;
+  transform: translate(-50%, -50%);
+  z-index: 9999;
+  background-color: #fff;
+  padding: 20px;
+  box-shadow: 1px 20px 40px 20px rgb(102, 90, 90);
```

Comment:
- [ ] As required by the previous reviewer 👇🏻 Kindly address this.
>  The margin between the buttons is now too wide as well. Please adjust it to look just like the Figma design 💯 👍🏻 


![image](https://user-images.githubusercontent.com/79058364/218288153-6fca5d2a-6a41-4817-9122-f92529f0a6a2.png)


### I can see that this is your last attempt 🤔 I'm going to approve this but I strongly recommend you fix this 💯 👍🏻